### PR TITLE
Create imnisarg.txt

### PR DESCRIPTION
**Pull request by imnisarg**
This Pull Request resolves Issue #<IssueNumber>

Description:
In this Pull Request we have done the following additions/edits for the code:

Note: Delete all the text including this!
* Make sure you replace `#<IssueNumber>` with the particular Issue that you are referencing with!
* Make sure you describe the Issue properly and ask as many people as possible to review your Pull Request so that it can be merged as soon as possible!

### CODE REVIEW

**Comment by imnisarg** [comment]
adding file

File: participants/imnisarg.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (pvgupta24) [inline participants/imnisarg.txt]: Add participant details here.

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR and label it as `spam`?

**Comment by salman-bhai** [comment]
spam!

**Pull request closed without merging**